### ISSUE TITLE
Output duration mismatch

### ISSUE THREAD

**Issue by dkrockcom** [fluent-ffmpeg/node-fluent-ffmpeg#887]
### Version information

* fluent-ffmpeg version: 2.1.2
* ffmpeg version: N-82966-g6993bb4
* OS: Windows 10

### Code to reproduce

```js
new ffmpeg({
	source: this.url
})
.duration(1800)
.videoBitrate(200)
.fps(this.dataRate)
.on('error', function (err, stdout, stderr) {
	console.log(stderr);
})
.on('start', function (evt) {
	console.log(evt);
})
.on('stderr', function (stderrLine) {
	console.log(stderrLine);
})
.on('end', function (stdout, stderr) {
	console.log(stderr, stdout);
}).format('mp4').save(filePath);

```

As I mentioned, in a sample code.

I have use this plugin for recording video from IP camera using RTSP URI with duration 30 Mins (1800 Sec).
and calling the recording function again after recording complete in my original code.
Example: If it is call 4-5 times after current record complete, so that.

### Expected results
1. Video_1 with 30:00 Mins Duration
2. Video_2 with 30:00 Mins Duration
3. Video_3 with 30:00 Mins Duration
4. Video_4 with 30:00 Mins Duration
5. Video_5 with 30:00 Mins Duration

### Observed results
1. Video_1 with 30:00 Mins Duration
2. Video_2 with 29:57 Mins Duration
3. Video_3 with 29:54 Mins Duration
4. Video_4 with 30:00 Mins Duration
5. Video_5 with 29:58 Mins Duration

**Comment by njoyard** [comment]
Have you tried directly with ffmpeg? Do you get reliable duration with it?

**Comment by dkrockcom** [comment]
> Have you tried directly with ffmpeg? Do you get reliable duration with it?

Yes but, there is some difficulty to maintain different-different command for different-different IP camera's and mobile camera.

That's why I use this plugin, please provide me a suitable solution for that.

Thanks,
Devesh
